themes/bootstrap5/js/searchbox_controls.js

```diff
@@ -238,6 +239,22 @@ VuFind.register('searchbox_controls', function SearchboxControls() {
           hiddenFilters.push($(this).val());
         });
 
+        if (applyActiveFilters) {
+          const queryParams = new URLSearchParams(window.location.search);
+          [...queryParams].forEach(
+            /**
+             * If the provided parameter is a filter, add it to the hiddenFilters array.
+             *
+             * @param {Array} queryParam Array representing query parameter ([key, value] format)
+             */
+            function extractFilters(queryParam) {
+              if (queryParam[0].startsWith("filter[")) {
+                hiddenFilters.push(queryParam[1]);
```

Comment:
One could argue that the `hiddenFilters` parameter used by the AJAX handler would be more appropriately renamed to `filters` since from the AC perspective, it doesn't really matter if they're hidden or not... but I opted not to change it here in the interest of minimizing changes.